apps/src/lib/kits/maker/dropletConfig.js

```diff
@@ -114,13 +114,13 @@ export const blocks = [
   // TODO(bbuchanan): Known issue - objectDropdown doesn't work with type:'readonlyproperty'
   {func: 'holdtime', objectDropdown: {options: BUTTON_VARS, dropdownOnly: true}, category: CIRCUIT_CATEGORY, blockPrefix: `${BUTTON_VARS[0]}.`, modeOptionName: "*.holdtime", type: 'readonlyproperty', tipPrefix: '[Button].'},
 
-  {func: 'soundSensor.start', category: CIRCUIT_CATEGORY},
+  {func: 'soundSensor.start', category: CIRCUIT_CATEGORY, noAutocomplete: true},
   {func: 'soundSensor.value', category: CIRCUIT_CATEGORY, type: 'readonlyproperty'},
   {func: 'soundSensor.getAveragedValue', category: CIRCUIT_CATEGORY, params: ['500'], paletteParams: ['ms'], type: 'value'},
   {func: 'soundSensor.setScale', category: CIRCUIT_CATEGORY, params: ['0', '100'], paletteParams: ['low', 'high']},
   {func: 'soundSensor.threshold', category: CIRCUIT_CATEGORY, type: 'property' },
 
-  {func: 'lightSensor.start', category: CIRCUIT_CATEGORY},
+  {func: 'lightSensor.start', category: CIRCUIT_CATEGORY, noAutocomplete: true},
```

Comment:
That just means it's knowledge-share time!  Allow me to give you a tour of these changes.

Here I've modified entries in the `maker/dropletConfig.js` `blocks` export.  We have several `dropletConfig.js` files for different apps and different APIs - for example, [here's the App Lab droplet config](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/applab/dropletConfig.js).  These are our own internal format, but many of these options get passed down to the [droplet editor](https://github.com/droplet-editor/droplet) used by App Lab, Game Lab, etc. These configuration files are generally responsible for:

- Providing a set of "blocks" the droplet editor will recognize, with various predefined block properties.
- Defining categories for the toolbox.
- Listing other 'predefined' values that affect linting within the editor.
- A few other configuration options (`showExamplesLink` or `showParamDropdowns`).

Note that this does _not_ determine which blocks appear in the level's toolbox, although this is a prerequisite for it - that happens later.  Nor does this config define the _behavior_ of these blocks, which is an interpreter concern more than an editor concern.

Here we're modifying the configuration of a couple of blocks.  Block configuration options [are documented in dropletUtils.js](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/dropletUtils.js#L3-L26).  The `noAutocomplete` option that I'm setting here reads "Do not include this function in our ace completer."  [Ace](https://ace.c9.io/) is the text editor that powers Droplet's text mode.  There's no autocomplete in block mode, for obvious reasons.

The droplet config for Maker Toolkit, since it's not associated with a particular app type, [gets "mixed in" to the App Lab droplet configuration](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/applab/applab.js#L494-L494) during App Lab initialization.  We do this even when Maker is disabled, so that the droplet representation of Maker code is always the same.